### ISSUE TITLE
[BUG] albert small google 版本在文本分类任务中展示出了极低的准确率

### ISSUE THREAD

**Issue by natureLanguageQing** [BrikerMan/Kashgari#346]
**You must follow the issue template and provide as much information as possible. otherwise, this issue will be closed.
请按照 issue 模板要求填写信息。如果没有按照 issue 模板填写，将会忽略并关闭这个 issue**
## Check List

Thanks for considering to open an issue. Before you submit your issue, please confirm these boxes are checked.
![image](https://user-images.githubusercontent.com/37429336/77432125-9e295200-6e18-11ea-9f3b-9f311a8b834d.png)
在bert cnn 中 第一轮没结束当前数据集的准确率已经接近了百分之百。
![image](https://user-images.githubusercontent.com/37429336/77432225-bdc07a80-6e18-11ea-92a6-3690cf0b1b4c.png)

但是 albert cnn中准确率缺低的可怜
**You can post pictures, but if specific text or code is required to reproduce the issue, please provide the text in a plain text format for easy copy/paste.**

- [ ] I have searched in [existing issues](https://github.com/BrikerMan/Kashgari/issues?utf8=%E2%9C%93&q=is%3Aissue+) but did not find the same one.
- [ ] I have read the [documents](https://kashgari.bmio.net)

## Environment

- windows
- Python Version: 3.7
- requirements.txt: 和官方一致 kashgari 1.1.1版本

```txt
[Paste requirements.txt file here]
```

### What
在运行过程中发现albert cnn的代码发生了准确率和召回率双底的情况
代码如下
    train_features, train_labels, valid_features, valid_labels, test_features, test_labels = read_message(
        "train.csv")
    model_folder = '../albert_small_zh_google'
    checkpoint_path = os.path.join(model_folder, 'albert_model.ckpt')
    config_path = os.path.join(model_folder, 'albert_config_small_google.json')
    vacab_path = os.path.join(model_folder, 'vocab.txt')
    embed = BERTEmbeddingV2(vacab_path,
                            config_path,
                            checkpoint_path,
                            bert_type='albert',
                            task=kashgari.CLASSIFICATION,
                            sequence_length=100)
    # 获取bert字向量
    model = CNNModel(embed)
    # 输入模型训练数据 标签 步数
    model.fit(train_features,
              train_labels,
              valid_features,
              valid_labels,
              epochs=8,
              batch_size=16,
              callbacks=[tf_board_callback]
              )
    # 保存模型
    model.evaluate(test_features, test_labels)
    model.save("output/albert_CNN-model")

[Tell us about the issue]

### Reproduce

[The steps to reproduce this issue. What is the URL you were trying to play, where did you put your code, etc.]

### Other Comment

[Add anything else here]

**Comment by BrikerMan** [comment]
这个应该和 #354 一个问题，已修复推送到 master 分支，麻烦测试一下。

```
pip uninstall kashgari && pip install git+https://github.com/BrikerMan/Kashgari
```

**Comment by natureLanguageQing** [comment]
`import os
import os
import kashgari
from kashgari.embeddings import BERTEmbeddingV2
from kashgari.tokenizer import BertTokenizer
import pandas as pd
import numpy as np
kashgari.config.use_cudnn_cell = True
model_folder = 'electra_tiny'

checkpoint_path = os.path.join('model.ckpt-1000000')
config_path = os.path.join('bert_config_tiny.json')
vocab_path = os.path.join('vocab.txt')

def data_doubles(train_data):
    train_features = [list(i[0]) for i in train_data]
    train_labels = [i[1] for i in train_data]
    return train_features, train_labels



def read_message(path):
    data = pd.read_csv(path).values.tolist()
    random_order = list(range(len(data)))
    np.random.shuffle(random_order)
    train_data, valid_data, test_data = [], [], []
    for i, j in enumerate(random_order):
        if i % 10 != 0 and i % 10 != 1:
            train_data.append(data[i])
        if i % 10 == 0:
            valid_data.append(data[i])
        if i % 10 == 1:
            test_data.append(data[i])
    train_features, train_labels = data_doubles(train_data)
    valid_features, valid_labels = data_doubles(valid_data)
    test_features, test_labels = data_doubles(test_data)
    return train_features, train_labels, valid_features, valid_labels, test_features, test_labels
train_features, train_labels, valid_features, valid_labels, test_features, test_labels = read_message("five_CLS_data.csv")

feature_list = []
label_list = []
max_len = 0
for i in five_CLS_data:
    feature_list.append(i[0])
    if len(i[0]) > max_len:
        max_len = len(i[0])
    label_list.append(i[1])
tokenizer = BertTokenizer.load_from_vacab_file(vocab_path)
embed = BERTEmbeddingV2(vocab_path,
                        config_path,
                        checkpoint_path,
                        bert_type='electra',
                        task=kashgari.CLASSIFICATION,
                        sequence_length=max_len)




from kashgari.tasks.classification import CNNLSTMModel

model = CNNLSTMModel(embed)

# ------------ build model ------------
model.fit(
    train_features, train_labels,
    valid_features, valid_labels,
    epochs=30,
    batch_size=32
)
`
结果

测试结果分数比较符合数据集应有的表现，感谢BrikerMan的每一次付出

300/300 [==============================] - 9s 30ms/step - loss: 0.1708 - acc: 0.9452 - val_loss: 0.2597 - val_acc: 0.9224
